src/actions.ts

```diff
@@ -3,6 +3,7 @@ import { getValue, getElement } from './transformers';
 import { po } from '@qavajs/po-playwright';
 import { expect } from '@playwright/test';
 import { parseCoords, parseCoordsAsObject } from './utils/utils';
+import memory from "@qavajs/memory";
```

Comment:
do not use memory directly, there is getValue function that already imported to file

Reply:
Resolved in https://github.com/qavajs/steps-playwright/pull/69/commits/d6dab87c19abe192da6c52b1761293b48f589000